src/Symfony/Component/AssetMapper/ImportMap/RemotePackageDownloader.php

```diff
@@ -69,11 +72,15 @@ public function downloadPackages(?callable $progressCallback = null): array
             return [];
         }
 
-        $contents = $this->packageResolver->downloadPackages($remoteEntriesToDownload, $progressCallback);
+        try {
+            $contents = $this->packageResolver->downloadPackages($remoteEntriesToDownload, $progressCallback);
+        } catch (TimeoutException|TransportException $exception) {
+            throw new LogicException($exception->getMessage());
+        }
```

Comment:
I don't think this is a good idea, as it's hiding the real exception. Also, this is not a "Logic" exception.